README.md

```diff
@@ -47,16 +47,12 @@ sbt test
 ```
 below command will help to run unit tests, integration tests and coverage 
 ```
-./check.sh
-```
-this can also be done with below command
-```
-sbt clean scalastyle coverage test coverageReport
+./run_all_tests.sh
 ```
 
 ### Journey tests and prototype
```

Comment:
### Journey tests and prototype
Please run the app with mocked address lookup for journey tests with script:
```
./run_for_journey_tests.sh
```

Reply:
I cant add it with proper format so here we go :D
![image](https://user-images.githubusercontent.com/11138469/189309438-b36a05f5-ead7-43b0-971c-78049137888b.png)